### PR TITLE
Add "ignore" to bower.json

### PR DESCRIPTION
**Pull request by FluorescentHallucinogen**

### CODE REVIEW

**Comment by FluorescentHallucinogen** [comment]
@valdrinkoshi, @azakus, please merge this pull request.

**Comment by FluorescentHallucinogen** [comment]
@notwaldorf, please merge this pull request. Sorry for pull requests spamming.

File: bower.json [hunk 1]
```diff
@@ -18,7 +18,18 @@
     "shim"
   ],
   "license": "BSD-3-Clause",
-  "ignore": [],
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "src",
```
Comment (valdrinkoshi) [inline bower.json]: 'src' should not be excluded.

Comment (FluorescentHallucinogen) [inline bower.json]: What about `gulpfile.js`, `package.json` and `tests`?

Comment (valdrinkoshi) [inline bower.json]: we can keep also `gulpfile.js, package.json`, tests are not needed

Comment (FluorescentHallucinogen) [inline bower.json]: Done.

Comment (redbugz) [inline bower.json]: Why send down gulpfile.js and package.json in the bower directory? What value do they provide to the consuming module?

**Comment by FluorescentHallucinogen** [comment]
@notwaldorf, @valdrinkoshi, @azakus Any progress?

File: bower.json (hunk 1 above)
Comment (FluorescentHallucinogen) [inline bower.json]: @valdrinkoshi WDYT?

Comment (stramel) [inline bower.json]: Typically, I see the gulpfile passed down so you can modify and run your own for easy testing but isn't necessary. Package.json would be useful when everyone starts migrating to yarn

**Comment by TimvdLippe** [comment]
Since we no longer have `bower.json`, I am closing this PR.

**Pull request closed without merging**